docs/contents.rst

```diff
@@ -4,70 +4,43 @@
    
    index
    mlops-dev-flow
-   install
    tutorial/index
-   
+   install   
 
 .. toctree::
    :maxdepth: 1
-   :caption: Concepts
+   :caption: Core components
       
    projects/project
-   concepts/data-feature-store
-   runtimes/functions
-   runtimes/load-from-marketplace
-   concepts/jobs
+   runtimes/serverless-functions
+   data/data-feature-store
    feature-store/feature-store
-   concepts/model-training
+   projects/workflows
    serving/serving-graph
-   concepts/model-serving
-   concepts/multi-stage-workflows
-   concepts/deployment-monitoring
-   concepts/ci-cd-automation
-  
-.. toctree::
-   :maxdepth: 1
-   :caption: Data Collection and Preparation
-   
-   store/datastore
-   data-collect-prep/ingesting_data
-   data-collect-prep/logging_datasets
-   feature-store/feature-store-data-ingestion
-   
+   model_monitoring/index
     
 .. toctree::
    :maxdepth: 1
-   :caption: Model Development and Testing
+   :caption: MLOps tasks
    
-   training/create-a-basic-training-job
-   training/working-with-data-and-model-artifacts
-   concepts/auto-logging-mlops
-   training/built-in-training-function
-   feature-store/training-fs   
-   hyper-params
-   projects/ci-integration
-  
+   data-collect-prep/ingest-process-data
+   model_development/continuous-model-development
+       
 .. toctree::
    :maxdepth: 1
-   :caption: Building Online ML Services
-
+   :caption: Deploy models and ML applications / online pipelines
```

Comment:
drop the `/ online pipelines` it was a 2nd option, just keep `Deploy models and ML applications`

Reply:
looks like its not the right hierarchy (should be 1 level down)